index.Rmd

```diff
@@ -40,79 +40,72 @@ can_lang <- readr::read_csv("https://github.com/ttimbers/canlang/raw/master/inst
   readr::write_csv("data/can_lang.csv")
 ```
 
-# R, Jupyter, and the tidyverse
+# R and the tidyverse
+## Overview
 
-This is an open source textbook aimed at introducing undergraduate students to data science. It was originally written for the University of British Columbia's [DSCI 100 - Introduction to Data Science](https://ubc-dsci.github.io/dsci-100/) course. In this book, we define data science as the study and development of reproducible, auditable processes to obtain value (i.e., insight) from data. 
+This textbook aims to be an approachable introduction to the world of data science. In this book, we define *data science* as the study and development of reproducible, auditable processes to obtain value (i.e., insight) from data. At a high level, in this book, readers will learn how to 
 
-The book is structured so that learners spend the first four chapters learning how to use the R programming language and Jupyter notebooks to load, wrangle/clean, and visualize data, while answering descriptive and exploratory data analysis questions. The remaining chapters illustrate how to solve four common problems in data science, which are useful for answering predictive and inferential data analysis questions:
+(1) use reproducible tools to do data analysis and 
+(2) identify and solve common problems in data science. 
 
-1. Predicting a class/category for a new observation/measurement (e.g., cancerous or benign tumour)
-2. Predicting a value for a new observation/measurement (e.g., 10 km race time for 20 year old females with a BMI of 25).
-3. Finding previously unknown/unlabelled subgroups in your data (e.g., products commonly bought together on Amazon)
-4. Estimating an average or a proportion from a representative sample (group of people or units) and using that estimate to generalize to the broader population  (e.g., the proportion of undergraduate students that own an iphone)
+The book is structured, so learners spend the first four chapters learning how to use the R programming language to load, wrangle, clean, and visualize data while answering descriptive and exploratory data analysis questions. The next six chapters illustrate how to solve common problems in data science, which help answer predictive and inferential data analysis questions. In the final chapter, we discuss how to set up your own computer with tools like R and Jupyter. For readers who need help setting up their system on their computer, we recommend starting with the [Setting up your own computer] chapter before reading chapter 2. Figure \@ref(fig:img-chapter-overview) summarizes a data science pipeline and what you will learn in each chapter of this book.
```

Comment:
We should formally cite the R programming language. The citation is pasted below and it should be put in `references.bib`. Once that is done we can reference it inline via `@Rlanguage`:

```
  @Manual{Rlanguage,
    title = {R: A Language and Environment for Statistical Computing},
    author = {{R Core Team}},
    organization = {R Foundation for Statistical Computing},
    address = {Vienna, Austria},
    year = {2021},
    url = {https://www.R-project.org/},
  }
```